src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -198,11 +198,11 @@ internal async Task ProcessDefaultInvocationRequest(ScriptInvocationContext scri
                     {
                         if (httpScriptInvocationResult.Outputs == null || !httpScriptInvocationResult.Outputs.Any())
                         {
-                            _logger.LogWarning("Outputs not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
+                            _logger.LogDebug("Outputs not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
```

Comment:
I was seeing these warnings because my handler doesn't return outputs. However, my handler doesn't require outputs, so I was confused by why a warning would be logged. It seemed to me that a warning was unnecessary, so I changed these to `LogDebug`. I'm open to an alternative approach if there's a strong reason to keep these as warnings.

Reply:
@pragnagopa may have context on why this warning was added, but I agree that it probably shouldn't be warning, and we probably shouldn't log anything at all since in your case this isn't a problem at all.

Reply:
Same as below - debug / trace level log is Ok for this.